substra/sdk/backends/local/hasher.py

```diff
@@ -14,4 +14,4 @@ def update(self, v):
         self._h.update(v)
 
     def compute(self):
-        return self._h.hexdigest()
+        return "local_" + self._h.hexdigest()
```

Comment:
IMO the "local_" keyword should be added outside of the hasher module. The hasher module should be only responsible for hashing stuff.

Reply:
I moved everything to DataAccess